ui/components/ui/popover/popover.component.js

```diff
@@ -135,8 +145,9 @@ const Popover = ({
             backgroundColor={BackgroundColor.backgroundDefault}
             color={Color.iconDefault}
             onClick={onScrollDownButtonClick}
-            className="whats-new-popup__scroll-button"
-            data-testid="whats-new-popup-scroll-button"
+            className="popover-scroll-button"
+            style={{ bottom: footer ? '140px' : '12px' }}
```

Comment:
I think we can add this style with `className` rather than a hardcoded bottom value. wdyt? Also, why are we using `140px `and `12px`. Is it used like this previously for the popover component?